recipes/coursier/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "coursier" %}
+{% set version = "2.0.7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://raw.githubusercontent.com/{{ name }}/{{ name }}/v{{ version }}/LICENSE
```

Comment:
In case the license does not change, you can simply upload it directly to the recipe repository.

Reply:
But in case it _does_? Granted, something that is Apache-2.0 is unlikely to change, but I guess I'd rather have it fail with a predictable `sums do not match` than just... not know, and keep shipping the wrong one.